README.md

```diff
@@ -0,0 +1,14 @@
+# Software Common Respository ![example workflow](https://github.com/SSL-A-Team/common/actions/workflows/CI.yml/badge.svg)
+
+Contains software artifacts shared across the firmware/software boundary.
+
+Sub folders contain relevant sub README files.
+ - radio/ - all common headers used in Robot\<-\>AI communications.
```

Comment:
I don't see a "radio" subdirectory in the actual repo...

Reply:
slipped through a refactor, fixed